src/pages/apis/drug/drugsfda/_infographic_definitions.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
@beardedfinch This file refers to Enforcement in a number of places.

Reply:
This file isn't used; I'll delete it.